resources/public/api/conf/1.0/schemas/amend_request_schema.json

```diff
@@ -0,0 +1,287 @@
+              "type":"number",
+              "example":"1999.99"
+            },
+            "taxPaid":{
+              "description":"The amount of tax to be paid on the lump sum benefit taken in excess of the lifetime allowance. The value must be between 0 and 99999999999.99 up to 2 decimal places.",
+              "type":"number",
+              "example":"1999.99"
+            }
+          },
+          "required": [
+            "amount",
+            "taxPaid"
+          ]
+        },
+        "benefitInExcessOfLifetimeAllowance":{
+          "type":"object",
+          "description":"Object holding the amount and tax paid on benefit in excess of the lifetime allowance.",
+          "properties":{
+            "amount":{
+              "description":"The amount of the benefit taken in excess of the lifetime allowance. The value must be between 0 and 99999999999.99 up to 2 decimal places.",
+              "type":"number",
+              "example":"1999.99"
+            },
+            "taxPaid":{
+              "description":"The tax to be paid on the benefit taken in excess of the lifetime allowance. The value must be between 0 and 99999999999.99 up to 2 decimal places.",
+              "type":"number",
+              "example":"1999.99"
+            }
+          },
+          "required": [
+            "amount",
+            "taxPaid"
+          ]
+        }
+      },
+      "required":[
+        "pensionSchemeTaxReference"
+      ]
+    },
+    "pensionSchemeOverseasTransfers":{
+      "type":"object",
+      "description":"Object holding transfer of overseas pension to another pension scheme.",
+      "properties": {
+        "overseasSchemeProvider": {
+          "description": "Details of overseas pension provider.",
+          "type": "array",
+          "items": {
+            "title": "item",
+            "type": "object",
+            "properties": {
+              "providerName": {
+                "description": "The name of the pension provider.",
+                "type": "string",
+                "example": "Overseas Pensions Plc"
+              },
+              "providerAddress": {
+                "description": "The address of the pension provider.",
+                "type": "string",
+                "example": "111 Main Street, Any Town, Any City"
+              },
+              "providerCountryCode": {
+                "description": "A three-letter code that represents a country name. The value must be a ISO 3166-1 alpha-3 code.",
+                "type": "string",
+                "example": "CYM"
+              },
+              "qualifyingRecognisedOverseasPensionScheme":{
+                "type":"array",
+                "Description": "A list of one or more qualifying recognised Overseas Pension Scheme reference number",
+                "items": {
+                  "description":"The qualifying overseas pension scheme reference number for overseas pensions contributions.",
```

Comment:
This isn't the same as the business spec - was it changed by Mashuk in his review? Business spec should reflect any changes made.